### DISCUSSION TITLE
如何在笔记工作区中显示光标？

### DISCUSSION THREAD

**Discussion by Leonoves** [windingwind/zotero-better-notes#722] [Q&A] [answered]
如题，在zotero better note中鼠标光标不再显示，也无法编辑文献链接如实现删除等功能，请问是设置的问题吗？图一是不显示光标的界面，图二是正常显示光标。
<img width="1920" alt="光标不显示" src="https://github.com/windingwind/zotero-better-notes/assets/85266230/bffe7b0e-ccb3-46ae-9488-a6b9c39a71f1">
![光标显示](https://github.com/windingwind/zotero-better-notes/assets/85266230/5caef463-0dc9-433c-bab5-bc837092afae)

**Comment by windingwind** [comment, marked as answer]
zotero 6 windows 版本问题。向zotero社区反馈，或升级到zotero7。与插件无关
